src/layer/Popup.js

```diff
@@ -179,7 +179,11 @@ export var Popup = DivOverlay.extend({
 			' leaflet-zoom-animated');
 
 		var wrapper = this._wrapper = DomUtil.create('div', prefix + '-content-wrapper', container);
-		this._contentNode = DomUtil.create('div', prefix + '-content', wrapper);
+		var wrapperContent = this._contentNode = DomUtil.create('div', prefix + '-content', wrapper);
+
+		wrapperContent.setAttribute('role', 'tooltip');

```

Comment:
> [`role="tooltip"`](https://www.w3.org/TR/wai-aria-1.1/#tooltip) doesn't seem appropriate for popups, rather [`role="dialog"`](https://www.w3.org/TR/wai-aria-1.1/#dialog).

_Originally posted by @Malvoz in https://github.com/Leaflet/Leaflet/pull/6248#r766351953_

Reply:
See issue https://github.com/Leaflet/Leaflet/issues/7968 for motivation for `role="dialog"`.